docker/db-init.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -euo pipefail
+echo "😃😃😃😃😃😃 DB SETUP TIME 😃😃😃😃😃"
+
+echo $POSTGRES_DB
+
+
+until PGPASSWORD=$POSTGRES_PASSWORD psql -h ${DOCKER_POSTGRES_HOST} -U $POSTGRES_USER -p $DOCKER_INTERNAL_PORT -c '\q'; do
```

Comment:
new issue idea: Could this all be moved into the `db` docker container? 
I usually see these kind of scripts within the container

Reply:
See my other comment. Hypothetically it could, but I don't really see it bringing a lot of value vs. getting mvp working